packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -165,13 +165,14 @@ abstract class TextSelectionControls {
     return delegate.copyEnabled && !delegate.textEditingValue.selection.isCollapsed;
   }
 
-  /// Whether the current [Clipboard] content can be pasted into the text field
-  /// managed by the given `delegate`.
+  /// Whether the text field managed by the given `delegate` supports pasting
+  /// from the clipboard.
   ///
   /// Subclasses can use this to decide if they should expose the paste
   /// functionality to the user.
+  ///
+  /// This does not consider the current contents of the clipboard.
```

Comment:
nit: could you elaborate a bit more on this? It confused me a little on the first read-through.

Reply:
Will do.